DataStorage.py

```diff
@@ -0,0 +1,99 @@
+from typing import Dict, List

```

Comment:
Please use `Any` instead of `object` in the type annotations. Basically the difference is that `Any` is saying "we don't know the type, please infer it", while `object` is saying "this is an object, don't infer anything else".
```suggestion
from typing import Any, Dict, List
```